### PR TITLE
get certs for ITs

### PR DESCRIPTION
**Pull request by mira-miracoli**
see:
- https://github.com/usegalaxy-au/infrastructure/pull/1769

### CODE REVIEW

File: sn06.yml [hunk 1]
```diff
@@ -37,17 +37,6 @@
       - "galaxyproject.eu"
       - "*.galaxyproject.eu"
       - "*.ep.interactivetool.usegalaxy.eu"
-      - "*.interactivetoolentrypoint.interactivetool.usegalaxy.eu"
-      - "*.interactivetoolentrypoint.interactivetool.live.usegalaxy.eu"
-      - "*.interactivetoolentrypoint.interactivetool.test.usegalaxy.eu"
-      - "*.aqua.usegalaxy.eu"
-      - "*.interactivetoolentrypoint.interactivetool.aqua.usegalaxy.eu"
-      - "*.ecology.usegalaxy.eu"
```
Comment (bgruening) [inline sn06.yml:44]: Here are the subsomains

Comment (mira-miracoli) [reply sn06.yml:44]: Are they still used, I think not because they are only a few and it should all be covered by the wildcard cert *.usegalaxy.eu

Comment (mira-miracoli) [reply sn06.yml:44]: I was wondering why there are so few, but maybe we only support these few subdomains for ITs?

Comment (bgruening) [reply sn06.yml:44]: Yes, we only supported a few.

**Review by bgruening — APPROVED** [review]

**Pull request merged**